jmx-ssl-config.yml.sample

```diff
@@ -0,0 +1,17 @@
+integration_name: com.newrelic.jmx
+
+instances:
+  - name: jmx
+    command: all_data
+    arguments:
+      jmx_host: jmx-host.localnet
+      jmx_port: 9999
+      jmx_user: admin
+      jmx_pass: admin
+      key_store: /etc/pki/JMXClientKeyStore.key
```

Comment:
Is it possible to use the default java truststore or does nrjmx need to be fed a truststore?

Reply:
yes, you can.. its optional. If you use the default truststore, any user can make the JMX query over SSL.

Reply:
Do we need to use the `OpenWithSSL` from the sdk to use that? Looks like you need all the key store and trust store variables set in order to use the ssl path.

Reply:
I could have changed Jmx.Open to use variadic arguments for Jmx.Open but I did not want to impact any older JMX integrations that use Jmx.Open
nrjmx (java) does not use the SSL arguments if its not supplied. nri-jmx wrapper uses Jmx.Open which has 4 arguments that I have left unchanged, so I don't impact other JMX integrations